sycl/source/detail/platform_impl.cpp

```diff
@@ -26,6 +27,50 @@ namespace detail {
 
 using PlatformImplPtr = std::shared_ptr<platform_impl>;
 
+// Return true if the given device 'Dev' matches with any filter
+bool isForcedDevice(detail::device_filter_list *FilterList,
```

Comment:
Minor. Suggest renaming to something like "isEnabledDevice" or reverse the logic and call it "isHiddenDevice"